tools/wagmi-example/src/components/Profile.tsx

```diff
@@ -0,0 +1,57 @@
+import {useAccountBalance} from "../hooks/use-balance.ts";
+import {useAccount, useConnect, useDisconnect} from "wagmi";
+
+export function Profile() {
+  const {status, address} = useAccount()
+  const {disconnect} = useDisconnect()
+  const {balance} = useAccountBalance();
+  const {connectors, connect, status: statusWalletConnect, error: walletError} = useConnect()
+
+  const isLoading = statusWalletConnect === "pending" || status === "connecting" || status === "reconnecting";
+
+  return (
+    <div>
+      {isLoading && (
+        <div>
+          <p>Loading...</p>
+        </div>
+      )}
+
+      {
+        status === "connected" && (
+          <div>
+            <h2>Account</h2>
+
+            <div>
+              <p>status: {status}</p>
+              <p>addresses: {JSON.stringify(address)}</p>
```

Comment:
Why do we need this stringify?

It should already be either null or a string at this point
https://1.x.wagmi.sh/react/hooks/useAccount#return-value